features/integration/evaldelta.feature

```diff
@@ -1,26 +0,0 @@
-Feature: EvalDelta
```

Comment:
This feature was only running on the Java SDK, and even on Java, it [hasn't been running since Feb 9, 2022](https://github.com/algorand/java-algorand-sdk/pull/290/files).